### PR TITLE
Fix/sponsors

### PR DESCRIPTION
**Pull request by zoek1**
##### Description

- [x] clicking on profile link should open in a new tab for the sponsor
- [x] As a hackathon sponsor when I click on the GitHub cat - the GitHub project should open in a new tab
- [x] As a hackathon sponsor when I click on the Project link, the link should not 404. It should open the project link iii a new tab.
- [x] As a hackathon sponsor when I hover over the prize values, nothing should happen. Please remove hover states
- [x] as a Hackathon sponsor when I mark a winner - the pay button should appear next to the winner column. Clicking pay takes the sponsor to the normal payout flow in a new tab. Once sponsor's transaction goes through, then the view should update as paid with date on the submission view. See mockup.
- [x]  clicking chat icon opens the chat window but it's displaying 404. Chat doesn't open the conversation with that user.
- [x]  As a hackathon sponsor when I click the name of my prize - the prize details should open in a new tab.

image

- [x]  Change Submission column header to Projects
- [x]  Organize projects by most recently submitted at the top
- [x]  For projects that haven't been submitted give a light gray background #F3F3F3 and a note that says Not Submitted
- [x]  On the Add Note - add hint text inside the input field "this note can only be viewed by you"

https://www.loom.com/share/3522b328cd434198a80cf9cdb84c8fec
##### Refers/Fixes
https://github.com/gitcoinco/web/issues/7158

### CODE REVIEW

**Comment by zoek1** [comment]
@thelostone-mc  @PixelantDesign https://www.loom.com/share/3522b328cd434198a80cf9cdb84c8fec

**Comment by frankchen07** [comment]
from the short jam with @PixelantDesign & @vs77bb today:

1. I think it's a little confusing with the dual locations above and below. Isn't it just submitted work, or do we want to see started and submitted work?

Screen Shot 2020-08-04 at 15 09 02

2. Contacts and winners are nice, but it seems like it takes up alot of "space". Every row has a contact bubble - is there a way to coalesce some of those elements into the view somehow?

Screen Shot 2020-08-04 at 15 12 44

3. I understand the note is personal right now, and is only viewed on this page, but it might not be explicit to someone who is using this page. Do we think it's **absolutely** necessary?

**Comment by frankchen07** [comment]
https://github.com/gitcoinco/web/pull/7176 addresses some of the comments above

**Comment by PixelantDesign** [comment]
Looks great  @zoek1!  Could we make a few more updates?

- [ ]  Change Submission column header to **Projects**
- [ ]  Organize projects by most recently submitted at the top
- [ ]  For projects that haven't been submitted give a light gray background #F3F3F3 and a note that says  _Not Submitted_
- [ ] On the Add Note - add hint text inside the input field "this note can only be viewed by you"

Screen Shot 2020-08-05 at 11 16 01 AM

File: app/dashboard/templates/dashboard/sponsors.html [hunk 1]
```diff
@@ -172,38 +176,33 @@
...
+                          <b-card-header header-tag="header" class="justify-content-between row mx-0 px-4 py-2 bg-white" role="tab">
 
                             <div class="col-12 col-md-5 col-lg-6 text-center text-md-left pl-md-0">
                               <div class="my-2">
                                 <p class="m-0">
                                   <a target="_blank" :href="prize.url" class="font-body font-weight-semibold d-inline-block text-black">
                                     [[`${prize.title.slice(0, 100)}${prize.title.length > 100 ? '...' : ''}`]]
                                   </a>
+                                  <span v-if="prize.value_eth" class="tag token inner-tooltip">[[prize.value_eth | decimals]] ETH</span>
+                                  <span v-if="prize.values_usdt" class="tag usd inner-tooltip">[[prize.values_usdt]] USD</span>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/sponsors.html:188]: ^ Zoek could you revert this to how this was ?
Cause we had issues where this token would break into multiple lines if the title was long

File: app/dashboard/templates/dashboard/sponsors.html [hunk 2]
```diff
@@ -172,38 +176,33 @@
...
+                              <p v-if="!prize.total_projects" class="font-body font-weight-semibold m-0 text-left">No Submissions</p>
 
                               <p v-else class="m-0 position-relative">
-                                <span>
+                                <span style="left: 0;" class="position-absolute">
                                   [[ prize.total_submitted ]] / [[prize.total_projects]] Submitted
                                 </span>
                                 <span v-if="prize.paid" class="ml-5 gc-text-green"><span class="fas fa-check mr-2"></span>PAID</span>
-                                <a v-else :href="`/issue/payout?pk=${prize.pk}`" class="ml-2 ml-xl-5 btn btn-outline-primary font-smaller-5">PAY</a>
+                                <a v-else :href="`/issue/payout?pk=${prize.pk}`" class="ml-5 gc-text-blue">UNPAID</a>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/sponsors.html:201]: Why was this changed back ? Could we keep it to what it was

File: app/dashboard/templates/bounty/fulfill.html [hunk 3]
```diff
@@ -83,6 +83,15 @@ <h4 class="text-center mb-4">{% trans "Submit Work" %}</h4>
...
                   </div>
 
+                  <div class="mt-2">
+                    <label class="form__label" for="projectId">{% trans "Select the submitting project" %}</label>
+                    <select name='projectId' id='projectId' class="form__input w-100 js-select2 font-body" required>
+                      {% for project in prize_projects %}
+                      <option value="{{ project.id }}">{{ project.id }} - {{ project.name }}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty/fulfill.html:93]: ^ why this there for every bounty 
Wrap it in a if check to check if there are prize projects please

File: app/dashboard/templates/dashboard/sponsors.html [hunk 4]
```diff
@@ -172,38 +176,33 @@
...
 
-                                <b-button style="right: 0;top: -5px" class="position-absolute" v-if="prize.total_projects" v-b-toggle="`accordion-${prizeIndex}`" variant="link">
-                                  <i class="fas fa-angle-up"></i>
+                                <b-button style="right: 0;top: -5px" class="position-absolute" v-if="prize.total_projects" v-b-toggle="`accordion-${prizeIndex}`" @click="togglePrize(prizeIndex)" variant="link">
+                                  <i class="fas fa-angle-up" v-if="!prize.open"></i>
+                                  <i class="fas fa-angle-down" v-else></i>
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/sponsors.html:205]: <i class="fas fa-angle-down" v-if="prize.open"></i>
<i class="fas fa-angle-up" v-else></i>

File: app/dashboard/templates/dashboard/sponsors.html [hunk 5]
```diff
@@ -256,12 +257,19 @@
                                         Winner
                                       </b-form-checkbox>
                                     </b-td>
+                                    <b-td :class="isMobileDevice ? 'no-border mt-2': 'py-4'">
```
Comment (thelostone-mc) [inline app/dashboard/templates/dashboard/sponsors.html:260]: how does is `isMobileDevice` btw ? when I shrink my screen -> none of these come into effect

File: app/dashboard/templates/dashboard/sponsors.html (hunk 2 above)
Comment (zoek1) [reply app/dashboard/templates/dashboard/sponsors.html:201]: @PixelantDesign request me change this to the previous implmementation

File: app/dashboard/templates/dashboard/sponsors.html (hunk 5 above)
Comment (zoek1) [reply app/dashboard/templates/dashboard/sponsors.html:265]: Thanks, fixed now :+1:

File: app/dashboard/templates/bounty/fulfill.html (hunk 3 above)
Comment (zoek1) [reply app/dashboard/templates/bounty/fulfill.html]: done :)

File: app/dashboard/templates/dashboard/sponsors.html (hunk 1 above)
Comment (zoek1) [reply app/dashboard/templates/dashboard/sponsors.html]: done :)

File: app/dashboard/templates/dashboard/sponsors.html (hunk 2 above)
Comment (thelostone-mc) [reply app/dashboard/templates/dashboard/sponsors.html:206]: @PixelantDesign waiting on confirmation ^

Comment (PixelantDesign) [reply app/dashboard/templates/dashboard/sponsors.html:206]: What  was the change y'all are referring to?

Comment (thelostone-mc) [reply app/dashboard/templates/dashboard/sponsors.html:206]: @zoek1 looks like it was never supposed to be a link in the first place!
If that is the case -> this makes sense but you'll have to remove the <a tag>

I leave it to you and alisa to decide if on clicking this label -> it behaves as an accordion 

image

**Comment by zoek1** [comment]
- [ ]  the alignment is off on the $ value, make it  closer to where it was
- [ ] when there is a submission …..change pay to review submission - but not  make it a link
- [ ] review submission expands the   rows
- [ ] github link should be under the  projext title.

**Comment by PixelantDesign** [comment]
yayyyy
